.github/workflows/build-macos.yml

```diff
@@ -30,59 +30,58 @@ jobs:
       OPT: ${{matrix.cfg.opt}}
     steps:
     - uses: actions/checkout@v4
-    - name: Cache compile
-      id: cache-compile
-      uses: actions/cache@v3
-      env:
-        cache-name: cache-keep-compile
-      with:
-        path: |
-          libs/openFrameworksCompiled/lib/osx/**/
-          addons/obj/osx/**/
-
-        # key: ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-${{ hashFiles('**/*.cpp') }}
-        key: ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-
-        restore-keys: |
-          ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-
-
+#    - name: Cache compile
+#      id: cache-compile
+#      uses: actions/cache@v3
+#      env:
+#        cache-name: cache-keep-compile
+#      with:
+#        path: |
+#          libs/openFrameworksCompiled/lib/osx/**/
+#          addons/obj/osx/**/
+#
+#        # key: ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-${{ hashFiles('**/*.cpp') }}
+#        key: ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-
+#        restore-keys: |
+#          ${{ runner.os }}-${{matrix.cfg.opt}}-${{ env.cache-name }}-
 
-    - name: Cache libs
-      id: cache-libs
-      uses: actions/cache@v3
-      env:
-        cache-name: cache-install-libs
-      with:
-        path: |
-          libs/boost/**
-          libs/cairo/**
-          libs/curl/**
-          libs/fmod/**
-          libs/FreeImage/**
-          libs/freetype/**
-          libs/glew/**
-          libs/glfw/**
-          libs/glm/**
-          libs/json/**
-          libs/pugixml/**
-          libs/rtAudio/**
-          libs/tess2/**
-          libs/uriparser/**
-          libs/utf8/**
-          addons/ofxAssimpModelLoader/libs/**/
-          addons/ofxKinect/libs/**/
-          addons/ofxOpenCv/libs/**/
-          addons/ofxSvg/libs/**/
-          addons/ofxPoco/libs/**/
-
-        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('scripts/apothecary/apothecary/formulas/**/*.sh') }}
```

Comment:
@dimitre I think for this to work correctly the apothecary submodule would need to be updated earlier in this script

Reply:
Glad you found the issue. maybe it was caching a previous version of apothecary libs.